examples/tutorials/details/nested_sampling_Crab.py

```diff
@@ -358,6 +359,45 @@
 
 # sphinx_gallery_thumbnail_number = 3
 
+######################################################################
+# Using the samples to draw the spectral model error band.
+# -----------------------
+#
+# While the spectral error band is usually computed from the
+# covariance matrix of the parameters, it can also be computed from the
+# samples of the posterior distribution.
+# This is a more robust way to compute the error band as it takes into
+# account the full posterior distribution and not just the covariance
+# matrix.
```

Comment:
Should mention that using the covariance matrix implicitly assumes gaussian errors while from the posterior samples we don't assume a shape for the distribution (as long as the prior are only weakly influential).